frame/ibc/ping/src/lib.rs

```diff
@@ -147,6 +140,21 @@ pub mod pallet {
 	}
 }
 
+impl<T: Config> Pallet<T> {
+	pub fn send_ping_impl(params: SendPingParams) -> Result<(), ibc_primitives::Error> {
```

Comment:
where's the suffix `impl` coming from?

Reply:
not sure what you mean?

Reply:
I was just wondering why the `_impl` on the name of the method - curiosity. Is it something we choose? Or are we following  a convention from ... ??

Reply:
oh `send_ping` is already used by the pallet call impl